docs/libs/status.md

```diff
@@ -41,7 +26,7 @@ updatedCons, changed := updater.Conditions()
 
 For simplicity, all commands can be chained:
 ```go
-updatedCons, changed := conditions.ConditionUpdater(func() conditions.Condition[bool] { return &conImpl{} }, oldCons, false).UpdateCondition("myCondition", true, "newReason", "newMessage").Conditions()
+updatedCons, changed := conditions.ConditionUpdater(oldCons, false).UpdateCondition("myCondition", conditions.FromBool(true), myObj.Generation, "newReason", "newMessage").Conditions()
```

Comment:
Why do we need to set `true` or `false` in the `ConditionUpdater` function? I've already read 

> The bool argument determines whether old conditions that are not updated remain in the returned list (`false`) or are removed, so that the returned list contains only the conditions that were touched (`true`)

... but isn't that to complex for our cases? What else do you have in mind? From my perspective we just want to overwrite the fields of the conditions that changed and for the diff we will fire an Event. Why then the quote above?

Reply:
I created the issue as discussed: #90